packages/twenty-front/src/modules/object-record/graphql/optimistic-effect-definition/getRecordOptimisticEffectDefinition.ts

```diff
@@ -11,22 +14,27 @@ export const getRecordOptimisticEffectDefinition = ({
   objectMetadataItem: ObjectMetadataItem;
 }) =>
   ({
-    key: `record-create-optimistic-effect-definition-${objectMetadataItem.nameSingular}`,
     typename: `${capitalize(objectMetadataItem.nameSingular)}Edge`,
     resolver: ({
       currentData,
       newData,
+      updatedData,
       deletedRecordIds,
+      variables,
     }: {
       currentData: unknown;
-      newData: { id: string } & Record<string, any>;
+      newData?: { id: string } & Record<string, any>;
```

Comment:
createdRecords

Reply:
I feel the API should support createdRecords to be an array to be consistent with updatedRecords and deletedRecordIds